tests/Set/DoctrineORM29Set/Fixture/one_to_many_to_attribute.php.inc

```diff
@@ -20,7 +20,7 @@ namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
 
 use Doctrine\ORM\Mapping\OneToMany;
 
-class Product
+class OneToManyToAttribute
```

Comment:
@alexander-schranz The best practise is to name test fixture by the file name, so it respect PSR-4.

There were ~5 classes named the same way `Product`, so the static autolaoder mixed up to one and only used the first one. That usually leads into WTF solutions, like one `Product` class has a property, the `Product` doesn't.